index.js

```diff
@@ -10,7 +10,7 @@ const run = async () => {
     const owner = github.context.repo.owner;
     const repo = github.context.repo.repo;
 
-    const { data: commits } = await octokit.rest.pulls.listCommits({
+    const commits = await octokit.paginate(octokit.rest.pulls.listCommits, {
```

Comment:
Just double checking, I noticed that we removed { data: commits }, do you think `await octokit.paginate` returns the commits directly?

Reply:
yep from their docs:

<img width="1132" alt="image" src="https://user-images.githubusercontent.com/2239732/195444510-99b02964-8e81-403f-936c-6de52dfe5acf.png">

I tried it out locally and it worked OK with this new variable.